pandasai/llm/gpt4all.py

```diff
@@ -0,0 +1,181 @@
+"""GPT4All LLMs
+This module provides a family of commercially / non-commercially available 
+LLMs maintained by Nomic AI. With GPT4All we can easily download models like 
+Falcon, Llama etc and run them seamlessly using CPU with lower RAMs. 
+
+Example:
+    Use below example to call GPT4All supporrted models
+
+    >>> from pandasai.llm.gpt4all import GPT4AllLLM
+"""
+import os
+import requests
+from typing import Optional
+
+from pandasai.prompts.base import Prompt
+from .base import LLM
+
+
+class GPT4AllLLM(LLM):
+    """GPT4All LLMs API
+    Base LLM class is extended to support GPT4AllLLM. When this class will be
```

Comment:
```suggestion
    Base LLM class is extended to support GPT4All LLM. When this class will be
```
It's rather a verbal description of a model to support, no need to match the class name exactly.

Reply:
Sounds good.